server/start.go

```diff
@@ -459,6 +466,7 @@ func startInProcess(ctx *server.Context, clientCtx client.Context, appCreator ty
 		}
 
 		logger.Info("Bye!")
+		return nil
 	}()
```

Comment:
Ref: https://stackoverflow.com/questions/57740428/handling-errors-in-defer and https://trstringer.com/golang-deferred-function-error-handling/

Reply:
@fedekunze nice, thank you :)